src/interface/CrossChainMessageInterface.sol

```diff
@@ -23,6 +23,6 @@ interface CrossChainMessageInterface {
     uint32 toChain,
     uint256 gasLimit,
     uint256 gasPrice,
-    uint256 payloadLength
+    bytes calldata crossChainPayload
```

Comment:
Also just curious why the change?

Reply:
To properly calculate the L1 Security Fee, we need entire payload data. Zero bytes and non-zero bytes have different costs associated with them.

Reply:
aha good to know